_pkgdown.yml

```diff
@@ -1,37 +1,47 @@
 url: https://insightsengineering.github.io/rtables
 
 template:
-  bootstrap: 5
-  bootswatch: flatly
+  package: nesttemplate
 
 navbar:
   type: default
   right:
     - icon: fa-github fa-lg
       href: https://github.com/insightsengineering/rtables
 
-
 articles:
-  - title: For Users
-    desc: Vignettes aimed at package users
+  - title: Getting Started
+    navbar: Getting Started
     contents:
     - introduction
     - clinical_trials
+
+  - title: Table Customization
+    navbar: Table Customization
+    contents:
+    - custom_appearance
+    - title_footer
+    - sorting_pruning
+    
+  - title: rtables Concepts
```

Comment:
Any ideas for this section name?

Reply:
I think rtables concepts is good! It mentions the articles that are specific to rtables computations so it's correct and informative.

Reply:
sounds good to me!